[DOCS] Adds the 8.1.0 release notes (#126718)
elastic/kibana: docs/CHANGELOG.asciidoc

Operations::
Improves the file logging capabilities so that missing directories in the configured file path are now created before {kib} attempts to write to the file {kibana-pull}117666[#117666]

Platform::
* Add a new `elasticsearch.compression` configuration property to enable compression for communications between {kib} and {es} {kibana-pull}124009[#124009]
* Adds support of comments {kibana-pull}122457[#122457]
* Adds support for PNG and PDF reports on Darwin Arm64 architecture {kibana-pull}122057[#122057]
* Short URL client is now accessible on the frontend through plugin contract. *Dashboard* and *Discover* shared short URLs now contain a three word, human-readable slug {kibana-pull}121886[#121886]
* Adds the ability to add URL drilldowns to *Dashboard* panels {kibana-pull}121801[#121801]
* Adds a new structure to the report details flyout to help you find information faster {kibana-pull}120617[#120617]
* Adds HTML tag and impact level to axe-core CI violation reporter {kibana-pull}119903[#119903]
* Exposes {es} accuracy warnings to the user {kibana-pull}116632[#116632]

Querying & Filtering::
Improves the version field type {kibana-pull}123739[#123739]

Security::

> Removes the audit space {kibana-pull}124145[#124145]

Reviewer: ```suggestion
* Audit logs now include records for individual saved objects when an entire space is deleted {kibana-pull}124145[#124145]
* User login audit events now include the session ID for better correlation, and single sign-on flows no longer result in an extra `user_logout` event {kibana-pull}124299[#124299]
```

* Running disabled rules are now skipped {kibana-pull}119239[#119239]

Canvas::
* Fixes an issue where the image repeat element was not updating {kibana-pull}118701[#118701]
* Fixes an issue where *Canvas* validated values before saving variables {kibana-pull}118694[#118694]

Dashboard::
Adds the listing page callout when new dashboards are in progress {kibana-pull}117237[#117237]

Discover::
* Adds the ability to close the expanded document sidebar when you change data views {kibana-pull}119736[#119736]
* Fixes search on page load tests {kibana-pull}119087[#119087]

Elastic Security::
For the Elastic Security 8.1.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].

Enterprise Search::
For the Elastic Enterprise Search 8.1.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].

Fleet::
* Readded missing packages to keep up to date list {kibana-pull}125787[#125787]
* Trimmed whitespace from package policy names {kibana-pull}125400[#125400]

Lens & Visualizations::

> * Extracted and injected references for by value in *Visualize Library* {kibana-pull}126499[#126499]

Reviewer: ```suggestion
*  Fixes some dashboard visualizations that could show "Could not located index pattern" errors when copied from one space to another {kibana-pull}126499[#126499]
```

I think this description would be better (I added it to the PR description as well) but I will defer to @flash1293

* Include error documents in fallback query for services {kibana-pull}123554[#123554]
* Rewrite the data fetching for Inventory Threshold rule {kibana-pull}123095[#123095]
* Optimizations for Inventory Threshold Alerts {kibana-pull}122460[#122460]
* Increase composite size to 10K for Metric Threshold Rule and optimize processing {kibana-pull}121904[#121904]
* Fixes missing EUI theme in context {kibana-pull}121796[#121796]
* Rename alerting types in triggers_actions_ui {kibana-pull}121107[#121107]
* Fixes loading message for correlations table {kibana-pull}120921[#120921]
* Prefer host.name over host.hostname {kibana-pull}119952[#119952]

Platform::
* Improve `bfetch` error handling {kibana-pull}123455[#123455]
* Fixes a CSV export Reporting issue where expensive queries were used to collect the data when they were not needed {kibana-pull}123412[#123412]
* Fixes URL drilldown placeholder text and add placeholder capability to Monaco {kibana-pull}121420[#121420]
* Consider expired tasks invalid {kibana-pull}119664[#119664]
* `columns`. Fixes Bugs caused by using name instead of ID {kibana-pull}118470[#118470]

Security::

> Adds session ID to user login audit event {kibana-pull}124299[#124299]

Reviewer: ```suggestion
User login audit events now include the session ID for better correlation, and single sign-on flows no longer result in an extra `user_logout` event {kibana-pull}124299[#124299]
```
